services/app-api/storage/table.ts

```diff
@@ -0,0 +1,62 @@
+import { convertToDynamoExpression } from "../handlers/dynamoUtils/convertToDynamoExpressionVars";
+import { StatusCodes } from "../utils/constants/constants";
+import dynamoDb from "../libs/dynamodb-lib";
+import * as Types from "../types";
+import { MeasureParameters } from "../types";
+
+export const putToTable = async (
+  tableName: string,
+  data: any,
+  key: any,
+  columns: any
+) => {
+  const params = {
+    TableName: tableName,
+    Key: { ...key },
+    ...convertToDynamoExpression(
+      {
+        lastAltered: Date.now(),
+        data,
+        ...columns,
+      },
+      "post"
+    ),
+  };
+  await dynamoDb.update(params);
+  return { status: StatusCodes.SUCCESS, body: params };
+};
+
+export const getMeasureFromTable = async (parameters: MeasureParameters) => {
+  const { state, year, coreSet, measure } = parameters;
+  return await dynamoDb.get<Types.Measure>({
+    TableName: process.env.measureTableName,
+    Key: {
+      compoundKey: `${state}${year}${coreSet}${measure}`,
+      coreSet: coreSet,
+    },
+  });
+};
+
+const coreSetGroup = {
+  ACS: [Types.CoreSetAbbr.ACSC, Types.CoreSetAbbr.ACSM],
+  CCS: [Types.CoreSetAbbr.CCSC, Types.CoreSetAbbr.CCSM],
+};
+
+export const getMeasureByCoreSet = async (
+  coreSet: "ACS" | "CCS",
+  pathParams: MeasureParameters
+) => {
+  const measures = [];
+  if (coreSet) {
+    const group = coreSetGroup[coreSet];
+
+    for (let i = 0; i < group.length; i++) {
+      const measure = await getMeasureFromTable({
+        ...pathParams,
+        coreSet: group[i],
+      });
+      measures.push(measure);
+    }
+  }
+  return measures;
+};
```

Comment:
Normally I'd suggest `for..of` in place of this `for`, but here I would even go a step further, as long as we are getting 2 measures. If it were more I'd want to call `dynamoDb.query`, and if it were fewer we wouldn't need this function at all.

The type system should guarantee that coreSetGroup is always defined, although I should note that I have not run this code:

```suggestion
export const getMeasureByCoreSet = async (
  combinedCoreSet: "ACS" | "CCS",
  params: MeasureParameters
) => {
  const gets = coreSetGroup[combinedCoreSet].map(
    (coreSet) => getMeasureFromTable({ ...params, coreSet })
  );
  return Promise.all(gets);
};
```